docs/docs/models/td-full-metrics.md

```diff
@@ -0,0 +1,162 @@
+# Time dependent model performance
+
+## Species model
+
+The model was trained using 249,269 videos 14 countries in West, Central, and East Africa.
+To evaluate the performance of the model, we held out 30,324 videos from 689 randomly-chosen transects.
+Importantly, all videos from each transect were either used for training or held out for evaluation.
+So the performance of the model on the holdout set should reflect its performance on videos from a transect
+the model has never seen.
+
+### Description of the holdout set
```

Comment:
In general I think this is super clear and well explained. One overall thought I would strongly suggest is updating the order of the ### sections in line with what we hear from users, which is almost in reverse. For example:

- Removing blank videos first, right below the Species model intro - we know this is something pretty much everyone cares about as one of the first things they are interested in
- Species accuracy
- Recall and precision by species
- Description of the holdout set (good to have, but don't think this needs to be the first thing users go through)